### PR TITLE
refresh

### PR DESCRIPTION
**Pull request by eprince1**
update names

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -11,14 +11,14 @@ axiosRetry(axiosInstance, {
...
-  forceUpdate: actions.forceUpdate,
+  forceRefresh: actions.forceRefresh,
   clearCache: actions.clearCache,
   clearLocalStorage: actions.clearLocalStorage,
   clearSessionStorage: actions.clearSessionStorage,
   clearCookies: actions.clearCookies
 };
 
-const prdUrl = 'https://us-central1-jellysync.cloudfunctions.net/api';
+const prdUrl = 'http://localhost:5001/jellysync/us-central1/api';
```
Comment (avisprince) [inline index.js]: did you mean to include this?

Comment (avisprince) [reply index.js]: honestly this needs to be a config variable

Comment (eprince1) [reply index.js]: damn, yeah

Comment (eprince1) [reply index.js]: i'll do that a different pr

**Review by avisprince — APPROVED** [review]

**Pull request merged**